cypress/integration/project/creation.spec.ts

```diff
@@ -289,36 +290,53 @@ context("project creation", () => {
     });
 
     context("happy paths", () => {
-      it("creates a new project from an empty directory", () => {
-        withEmptyDirectoryStub(() => {
-          commands.pick("new-project-button").click();
+      context("creates a new project from an empty directory", () => {
+        function go(expectedDefaultBranch: string) {
```

Comment:
Can we change the name of the function to something more descriptive, so that it reads better where it is used?

Reply:
So the function is defined within its `context`, so for me `go` is as descriptive as it gets without repeating the same thing again. Would `run` be better, to mean "run" the test? Or feel free to shoot other suggestions :)

Reply:
You are right, since it is within the context, the definition of the function is not far away :)
I was thinking something like: `testNewProjectCreationWith`